### DISCUSSION TITLE
Ordering To-Do List

### DISCUSSION THREAD

**Discussion by b1u3too** [rachelpr/LeafSmart#105] [General]
Suggestion: Let's flag some buckets of when to work on certain goals/milestones

ie. Phase 1: MVD Bugfixes
Phase 2: Deployment
Phase 3: Set up CI/CD 
Phase 4: THE WORLD (iterative improvements)

**Comment by rachelpr** [comment]
Phase 1 would include bug fixes, but also what I'm dubbing a "feature" – adding in responsible/mobile first design
